app/main.py

```diff
@@ -1,6 +1,20 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    cinema_bar = CinemaBar()
+    customers_objs = []
+    for customer in customers:
+        customer_obj = Customer(customer.get("name"), customer.get("food"))
+        cinema_bar.sell_product(customer_obj.food, customer_obj)
+        customers_objs.append(Customer(name=customer["name"],
+                                       food=customer["food"]))
```

Comment:
```suggestion
        customers_objs.append(customer_obj)
```

You have already initialized customer_obj)

Reply:
Didn't notice))
Done